week-01/assessment/assessment-01-capsule-hotel/src/CapsuleHotel.java

```diff
@@ -1,5 +1,168 @@
+        startup();
+
+
+        // // Create a Scanner object.
+        Scanner console = new Scanner(System.in);
+        // assign int to variable capsules available.
+        int capsulesAvailable = Integer.parseInt(console.nextLine());
+        // create an array to hold how many capsules will the user enter.
+        String[] capsules = new String[capsulesAvailable];
+        // display message include the length of capsules.
+        System.out.println("There are " + capsules.length + " unoccupied capsules ready to be booked.\n");
+
+        String userInput;
+
+        do {
+            MainMenu();
+            userInput = console.nextLine();
+            switch (userInput) {
+                case "1" -> checkIn(console, capsules);
+                case "2" -> checkOut(console, capsules);
+                case "3" -> viewGuests(console, capsules);
+                case "4" -> exit(console);
+                default -> System.out.println("We don't have that option please try again and pick from [1-4]: ");
+            }
+        } while (!userInput.equals("0"));
+
+
+    }
+
+    // void method to display the menu
+    public static void MainMenu() {
+        System.out.print("""
+
+                Guest Menu
+                ==========
+                1. Check In
+                2. Check Out
+                3. View Guests
+                4. Exit
+                Choose on option [1-4]:\s""");
+    }
+
+
+    // Void method to display the welcome message
+    // display the message ask the user to enter number of capsules.
+    public static void startup() {
+        System.out.println("Welcome to Capsule-Capsule.");
+        System.out.println("=".repeat(27));
+        System.out.print("Enter the number of capsules: ");
+
     }
+
+    // exit method take a Scanner parameter
+    // which make the user enter either
+    // y to exit the program
+    // n to stay and avoid the data erase.
+    public static void exit(Scanner console) {
+        System.out.println("""
+                Exit
+                ====
+                Are you sure you want to exit?
+                All data will be lost.
+                Exit [y/n]:""");
+        String exitInput = console.nextLine();
+        if (Objects.equals(exitInput, "n")) {
+
+            return;
+
+        } else {
+            System.out.print("Good Bye, See you soon!");
+            System.exit(0);
+        }
+
+    }
+
+
+    // Check In method
+    // display messages ask the user-name, Capsule number
+    public static void checkIn(Scanner console, String[] capsules) {
+        System.out.println("Guess Check In: ");
+        System.out.println("==============");
+        System.out.print("Guest Name: ");
+        String guestName = console.nextLine();
+        System.out.print("Capsule #[1-" + capsules.length + "]: ");
+        int index = Integer.parseInt(console.nextLine()) - 1;
+
+        while (capsules[index] != null) {
```

Comment:
Shouldn't this be an `if` statement? Why use a `while`?

Reply:
For some reason I was thinking about to keep while loop because I need to keep looping, but when you mentioned that I can use if statement I used it and  works fine.
Thank you so much for mentioning that.